pkg/apimachinery/discoverysummarizer/server/config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
moved to `cmd/kubernetes-discover/server`.
Its a sample config file for the server.